src/BaselineOfCalypso/BaselineOfCalypso.class.st

```diff
@@ -98,9 +98,6 @@ BaselineOfCalypso >> baseline: spec [
 			package: #'Calypso-SystemPlugins-Deprecation-Queries-Tests'
 			with: [
 				spec requires: #( #'Calypso-SystemPlugins-Deprecation-Queries' ) ];
-			package: #'Calypso-SystemPlugins-FileOut-Browser' with: [
```

Comment:
I see this package removed from the baseline but the code is not removed from the code base of Pharo. This package should be removed.

Reply:
Done in https://github.com/pharo-project/pharo/pull/17101/commits/a4cd1a5da020b2870d879db4d5a327020334f346